library/src/skins/default/src/sass/webcomponents/sakai-notifications/_sakai-notifications.scss

```diff
@@ -0,0 +1,24 @@
+// Define BEM structure classes
+.sakai-notifications {
+  &__banner-warn {

```

Comment:
What do you think about the idea of just no using nesting in our SASS? One of the benefits of BEM is that is easier to search for a rule that you have seen in the markup. SASS nesting takes that advantage away.

https://bradfrost.com/blog/post/sass-selectors-to-nest-or-not-to-nest/

Reply:
force of habit, I put it without nesting, which will be much clearer, really.